Library/Homebrew/cask/lib/hbc/artifact/uninstall_base.rb

```diff
@@ -230,11 +237,13 @@ def uninstall_trash(directives, expand_tilde = true)
         uninstall_delete(directives, expand_tilde)
       end
 
-      def uninstall_rmdir(directives, expand_tilde = true)
-        Array(directives[:rmdir]).flatten.each do |directory|
-          directory = self.class.expand_path_strings([directory]).first if expand_tilde
-          directory = self.class.remove_relative_path_strings(:rmdir, [directory]).first
-          directory = self.class.remove_undeletable_path_strings(:rmdir, [directory]).first
+      def uninstall_rmdir(directories, expand_tilde = true)
+        directories = Array(directories[:rmdir]).flatten
+        directories = self.class.expand_path_strings(directories) if expand_tilde
+        directories = self.class.remove_relative_path_strings(:rmdir, directories)
+        directories = self.class.expand_glob(directories)
+        directories = self.class.remove_undeletable_path_strings(:rmdir, directories)
```

Comment:
Seems like these four lines are identical to the ones in `uninstall_delete`, so let's extract them into (yet another) helper method.